client/component/LinePlot.tsx

```diff
@@ -85,10 +89,24 @@ function LinePlot(props: LinePlotProps) {
       </p>
     );
   };
+  const [mode, setMode] = useState<string>('pan');
+  const interactionsConfig = createInteractionsConfig(
+    mode as 'pan' | 'zoom' | 'selection'
+  );
 
   return (
     <>
       <Toolbar>
+        <ToggleGroup
+          role="radiogroup"
+          ariaLabel="mode"
+          value={mode}
+          onChange={setMode}
+        >
+          <ToggleGroup.Btn label="pan" value="pan" />
```

Comment:
The three categories are "pan & wheel zoom", "select to zoom" and "select region". The latter will be extended with different region types but start off with a rectangle.